src/CommunityToolkit.Maui.MediaElement/Services/MediaControlsService.android.cs

```diff
@@ -18,15 +18,20 @@
 namespace CommunityToolkit.Maui.Media.Services;
 
 [SupportedOSPlatform("Android26.0")]
-[Service(Exported = false, Enabled = true, Name = "communityToolkit.maui.media.services", ForegroundServiceType = ForegroundService.TypeMediaPlayback)]
+[Service(Exported = false, Enabled = true, Name = "CommunityToolkit.Maui.Media.Services", ForegroundServiceType = ForegroundService.TypeMediaPlayback)]
 class MediaControlsService : Service
 {
-	public const string ACTION_PLAY = "MediaAction.play";
-	public const string ACTION_PAUSE = "MediaAction.pause";
-	public const string ACTION_UPDATE_UI = "CommunityToolkit.Maui.Services.action.UPDATE_UI";
-	public const string ACTION_UPDATE_PLAYER = "CommunityToolkit.Maui.Services.action.UPDATE_PLAYER";
-	public const string ACTION_REWIND = "MediaAction.rewind";
-	public const string ACTION_FASTFORWARD = "MediaAction.fastForward";
+	public const string ActionPlay = "MediaAction.play";
+	public const string ActionPause = "MediaAction.pause";
+	public const string ActionUpdateUI = "CommunityToolkit.Maui.Services.action.UPDATE_UI";
+	public const string ActionUpdatePlayer = "CommunityToolkit.Maui.Services.action.UPDATE_PLAYER";
+	public const string ActionRewind = "MediaAction.rewind";
+	public const string ActionFastForward = "MediaAction.fastForward";
```

Comment:
Why did you change the pattern? Can you revert and follow our pattern

Reply:
Isn't PascalCasing the standard convention in C# to name consts?

Reply:
@pictos I've left some comments on some of my code changes.